client/routes/execution/helpers/map-timeline-events.js

```diff
@@ -124,24 +158,32 @@ export default function(historyEvents) {
       });
     } else if (e.eventType === 'SignalExternalWorkflowExecutionInitiated') {
       add({
-        id: 'extsignal' + e.eventId,
+        id: `extsignal${e.eventId}`,
         className: 'external-signal',
         eventIds: [e.eventId],
         start: moment(e.timestamp),
         ongoing: true,
         content: 'External Workflow Signaled',
-        details: summarizeEvents.SignalExternalWorkflowExecutionInitiated(e.details),
+        details: summarizeEvents.SignalExternalWorkflowExecutionInitiated(
+          e.details
+        ),
       });
     } else if (e.eventType === 'ExternalWorkflowExecutionSignaled') {
       const initiatedEvent = hash[`extsignal${e.eventId}`];
       if (initiatedEvent) {
         initiatedEvent.eventIds.push(e.eventId);
-        assignEnd(item, e.timestamp);
+        // TODO - code will break as item is not defined.
+        // assignEnd(item, e.timestamp);
```

Comment:
Linter was complaining about item not defined. Because I am not 100% sure what this code is suppose to be doing I have just commented it out to avoid browser crashing.